### ISSUE TITLE
6.2 update

### ISSUE THREAD

**Issue by Ank164** [daemitus/XIVComboPlugin#191]
Plugin currently doesn't work anymore after 6.2

**Comment by PrincessRTFM** [comment]
The `IsActionIdReplaceable` signature (`81 F9 ?? ?? ?? ?? 7F 35`) is broken, something must have changed it in the last patch

**Comment by PrincessRTFM** [comment]
Update: `E8 ?? ?? ?? ?? 84 C0 74 4C 8B D3` works.

**Comment by BleuMage** [comment]
If this plugin still works, is there a way to force enable it? I tried changing the DalamudApiLevel to 7, but it told me there was an error loading it.

**Comment by Grammernatzi** [comment]
> If this plugin still works, is there a way to force enable it? I tried changing the DalamudApiLevel to 7, but it told me there was an error loading it.

The plugin works if you change the ActionIdReplaceable address and fix the SourceID to SourceId name change in the combo cache. There isn't a way to get it to work without editing the code, at the moment.

**Comment by daemitus** [comment]
I'll be working on plugins tonight

**Comment by kaedys** [comment]
On the mechanical updates side, the two changes relevant to this plugin that I saw were:

* Dive/High Dive no longer auto-changes into Mirage Dive, so we probably need back the pre-6.1 combo that handled that.
* Minor Arcana now auto-converts to the relevant card play, and Crown Play has been removed, so that combo can vanish.

The change to Redraw doesn't affect any existing combos that I can see, but it does make #181 more relevant, since Redraw now guarantees a different seal from the currently drawn one (doesn't guarantee a seal you don't yet have, but it does mean you're guaranteed at least 2 seals per Astrodyne, unless you just fail to Redraw entirely, and you have a much higher chance of a third).  Which means a combo that automates Redraw if your seal is redundant is quite a bit stronger than it was prior to 6.2.
